### ISSUE TITLE
Error when loading MaskRCNN2Go

### ISSUE THREAD

**Issue by hiankun** [caffe2/models#66]

**Comment by GBagley** [comment]
I'm having exactly the same problem.  When I examine the model/fp32/model.pb file it looks like this:

version https://git-lfs.github.com/spec/v1
oid sha256:4bcdabf3da95da012922b85c52aa8508cf0b80b997ce26367811665b5dcf86d5
size 21840

Some searching online indicates that this may stem from an issue with unresolved merges and lfs in git, which can be resolved by the owner of the git package. If you could fix/update the files that would be great. I suspect that they are unusable as they are now.  Thanks!

**Comment by RickLogi** [comment]
Ran into the same issue.  FAIR also contributed the model to  MLperf https://mlperf.org/ as part of the edge benchmarking  https://github.com/mlperf/inference/tree/master/edge/object_segmentation/maskrcnn2go  .   I was able to get this model to run but had to modify run_eval.sh slightly to extra -- from  `python code/eval_seg_cpu.py -- \`   to `python code/eval_seg_cpu.py \`

**Comment by liangmuren** [comment]
Same problem, too

**Comment by mathpopo** [comment]
download new protobuf git ,entry into python folder ,python setup.py install ,but show new error

**Comment by mathpopo** [comment]
def ParseFromString(self, serialized):
    """Parse serialized protocol buffer data into this message.

    Like MergeFromString(), except we clear the object first.
    """
    self.Clear()
    return self.MergeFromString(serialized)


def _DecodeUnknownField(buffer, pos, wire_type):
  """Decode a unknown field.  Returns the UnknownField and new position."""

  if wire_type == wire_format.WIRETYPE_VARINT:
    (data, pos) = _DecodeVarint(buffer, pos)
  elif wire_type == wire_format.WIRETYPE_FIXED64:
    (data, pos) = _DecodeFixed64(buffer, pos)
  elif wire_type == wire_format.WIRETYPE_FIXED32:
    (data, pos) = _DecodeFixed32(buffer, pos)
  elif wire_type == wire_format.WIRETYPE_LENGTH_DELIMITED:
    (size, pos) = _DecodeVarint(buffer, pos)
    data = buffer[pos:pos+size].tobytes()
    pos += size
  elif wire_type == wire_format.WIRETYPE_START_GROUP:
    (data, pos) = _DecodeUnknownFieldSet(buffer, pos)
  elif wire_type == wire_format.WIRETYPE_END_GROUP:
    return (0, -1)
  else:
    raise _DecodeError('Wrong wire type in tag.')-----------------------------show this

  return (data, pos)

**Comment by mathpopo** [comment]
@RickLogi  YES , i replace the pb file  in  https://github.com/mlperf/inference/tree/master/edge/object_segmentation/maskrcnn2go .so should the model pb model is error!

**Comment by mathpopo** [comment]
https://github.com/mlperf/inference/blob/master/others/edge/object_segmentation/maskrcnn2go/model/fp32/model.pb
